lib/rails/generators/channel/templates/channel.coffee.erb

```diff
@@ -0,0 +1,3 @@
+App.<%= channel_name %> = App.cable.subscriptions.create "<%= channel_class %>",
```

Comment:
This will generate JavaScript that creates an error unless the user has already defined `App.cable = Cable.createConsumer(...)`, and redeclaring a global `@App` in every file isn't ideal for those who don't wish to use that namespace.

Perhaps we should standardize defining your consumer and subscriptions on `@Cable` itself. Something like:

```coffee
@Cable =  
  consumer: null
  subscriptions: {}
```

And the generator could generate something like:
```coffee
Cable.subscriptions.<%= channel_name %> = Cable.consumer?.subscriptions.create "<%= channel_class %>", ...
```